it_zwierzak/homepage/views.py

```diff
@@ -1,23 +1,24 @@
 from django.shortcuts import render, redirect
 from django.contrib.auth.forms import UserCreationForm
 
+from django.contrib.auth import authenticate, login, logout
 
-def login(request):
-    context = {}
-    return render(
-        request,
-        'homepage/login.html',
-        context
-    )
+from django.contrib import messages
+
+from .forms import CreateUserForm
 
 
 def register(request):
-    form = UserCreationForm
+    form = CreateUserForm()
 
     if request.method == 'POST':
-        form = UserCreationForm(request.POST)
+        form = CreateUserForm(request.POST)
         if form.is_valid():
             form.save()
+            user = form.cleaned_data.get('username')
```

Comment:
Trochę mylące, bo jednak wyciągamy username, a nie usera
```suggestion
            username = form.cleaned_data.get('username')
```